vet_tests/test_wavelengths.py

```diff
@@ -78,3 +87,51 @@ def test_mdm_modspec(redux_out):
         # Load
         waveCalib = WaveCalib.from_file(file_path)
         assert waveCalib.wv_fits[index].rms < rms, f'RMS of mdm_modspec {setup} is too high!'
+
+def test_redoslits_kastr(redux_out):
+    """ Test the redo_slits option using shane_kast_red
+
+    Args:
+        redux_out (str): path to REDUX_OUT
+    """
+    setup = '600_5000_d46'
+
+    rdx_dir = os.path.join(redux_out,
+                             'shane_kast_red',
+                             setup)
+    # Artificially make the slit bad
+    slit_file = os.path.join(rdx_dir,
+                             'Calibrations',
+                             'Slits_A_0_DET01.fits.gz')
+    slits = SlitTraceSet.from_file(slit_file)
+    slits.mask[0] = slits.bitmask.turn_on(slits.mask[0], 'BADWVCALIB')
+    slits.to_file(slit_file, overwrite=True)
```

Comment:
I'm worried about overwriting the file.  Can we instead:
 - copy the original file
 - artificially make the slit bad in the original file
 - perform the test
 - delete the original (altered file) and move the copy back to the original file name

Reply:
done